pkgs/os-specific/linux/rtl8852bu/default.nix

```diff
@@ -56,11 +58,14 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
-  meta = with lib; {
+  meta = {
     description = "Driver for Realtek rtl8852bu and rtl8832bu chipsets, provides the 8852bu mod";
-    homepage = "https://github.com/morrownr/rtl8852bu";
-    license = licenses.gpl2Only;
-    platforms = platforms.linux;
-    maintainers = with maintainers; [ lonyelon ];
+    homepage = "https://github.com/morrownr/rtl8852bu-20240418";
+    license = lib.licenses.gpl2Only;
+    platforms = lib.platforms.linux;
```

Comment:
I think we're gonna need to do the same as in https://github.com/NixOS/nixpkgs/pull/410672/commits/79c1cf6a24aeea4d8195263035961c1d23627ebb